### PR TITLE
Made changes to parent pom so it now uses dependencyManagement

### PR DESCRIPTION
**Pull request by nazarjaber**
Started using dependencyManagement in parent pom

### CODE REVIEW

File: schema/target/src/main/java/generated/Dummy.java [hunk 1]
```diff
@@ -1,24 +1,8 @@
-/*
```
Comment (andyholst) [inline schema/target/src/main/java/generated/Dummy.java:1]: Generated Java classes from the XSD are not going to be merged.

File: schema/target/src/main/java/generated/ObjectFactory.java [hunk 2]
```diff
@@ -1,24 +1,8 @@
-/*
```
Comment (andyholst) [inline schema/target/src/main/java/generated/ObjectFactory.java:1]: Generated Java classes from the XSD are not going to be merged.

File: schema/target/src/main/java/se/jsquad/client/info/AccountTransactionApi.java [hunk 3]
```diff
@@ -1,24 +1,8 @@
-/*
```
Comment (andyholst) [inline schema/target/src/main/java/se/jsquad/client/info/AccountTransactionApi.java:1]: Generated Java classes from the XSD are not going to be merged.

File: schema/target/src/main/java/se/jsquad/client/info/AccountTransactionApi.java [hunk 4]
```diff
@@ -1,24 +1,8 @@
...
- * Copyright 2019 JSquad AB
```
Comment (MrAsimov) [inline schema/target/src/main/java/se/jsquad/client/info/AccountTransactionApi.java]: Same as above comment

File: schema/target/src/main/java/se/jsquad/client/info/TransactionTypeApi.java [hunk 5]
```diff
@@ -1,24 +1,8 @@
-/*
```
Comment (andyholst) [inline schema/target/src/main/java/se/jsquad/client/info/TransactionTypeApi.java:1]: Generated Java classes from the XSD are not going to be merged.

**Comment by andyholst** [comment]
The branch is not synced with the master branch, it's missing the travis yml pipeline file, so the tests are not verified if they succeed. Rebase with master branch and do a new branch commit.

**Review by andyholst — CHANGES_REQUESTED** [review]
Rebase with latest master commit (fbcf8c9a35298b2f714eea11664ddefcd81e25e7) and resolve the merge conflicts and verify the that the unit/system and integration executed tests is a success in Travis CI environment https://travis-ci.org/jsquad-consulting/openbank-jee.

**Pull request closed without merging**